게시판 만들기.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+
+class Post:
+    post_list = []
+
+    def __init__(self, title, body):
+        self.title = title
+        self.body = body
+
+    @classmethod
+    def add_post(cls):
+        title = input("추가할 글의 제목을 입력 : ").rstrip()
+        body = input("추가할 글의 본문을 입력 : ").rstrip()
+        if not (title and body):
+            print("[!] 제목과 본문은 공백일 수 없습니다!")
+        if not (type(title) == type(body) == type('str')):
+            print("[!] 제목과 본문은 문자열이여야 합니다!")
+
+        new_post = cls(title, body)
+        cls.post_list.append(new_post)

```

Comment:
함수의 범용성을 높이기 위해, `title`, `body`는 함수의 인자로 받아오는 게 맞을 것 같습니다. input받는 부분은 함수 호출 전에 진행하구요.

```suggestion
    def add_post(cls, title, body):
        if not (title and body):
            print("[!] 제목과 본문은 공백일 수 없습니다!")
        if not (type(title) == type(body) == type('str')):
            print("[!] 제목과 본문은 문자열이여야 합니다!")

        new_post = cls(title, body)
        cls.post_list.append(new_post)
```